drivers/clock_control/clock_control_nrf2_hfxo.c

```diff
@@ -68,13 +88,42 @@ static void onoff_start_hfxo(struct onoff_manager *mgr, onoff_notify_fn notify)
 	k_timer_start(&dev_data->timer, dev_config->start_up_time, K_NO_WAIT);
 }
 
+static void release_hfxo(struct dev_data_hfxo *dev_data)
+{
+	unsigned int key;
+
+	key = irq_lock();
+	if (dev_data->request_count < 1) {
```

Comment:
since `request_count` is unsigned `== 0` would be better as `< 1` might be misleading, indicating that it can be negative.

Reply:
I prefer avoid using sharp comparison.

Reply:
left `<1`